src/App.scss

```diff
@@ -1,8 +1,49 @@
+/* eslint-disable max-len */
+
 iframe {
   display: none;
 }
```

Comment:
1. use class selectors instead of tag selectors (you could only if there is no other way)
2. I don't see you use iframe in you JSX, why do you need to style it?

Reply:
fixed